src/components/Proposals/ProposalDetails/VoteActions.js

```diff
@@ -0,0 +1,220 @@
+import React, { useState, useCallback } from 'react'
+import styled from 'styled-components'
+import {
+  Button,
+  GU,
+  IconCheck,
+  IconConnect,
+  IconCross,
+  Info,
+  noop,
+  RADIUS,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { dateFormat, toMs } from '../../../utils/date-utils'
+
+import { useWallet } from '../../../providers/Wallet'
+
+export const VOTE_YEA = 'VOTE_YEA'
+export const VOTE_NAY = 'VOTE_NAY'
+const EMPTY_SCRIPT = '0x00000001'
+
+/* eslint-disable react/prop-types */
+const VoteActions = React.memo(({ vote, onVoteYes, onVoteNo, onExecute }) => {
```

Comment:
We should provide propTypes for this top level  component, feel free to disable the controlled props on internal elements though!